src/mscorlib/shared/System/Guid.Windows.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    partial struct Guid
+    {
+        // This will create a new guid.  Since we've now decided that constructors should 0-init,
+        // we need a method that allows users to create a guid.
+        public static Guid NewGuid()
+        {
+            // CoCreateGuid should never return Guid.Empty, since it attempts to maintain some
```

Comment:
I looked at CoCreateGuid and I do not see any place it checks for known GUIDs.

Reply:
This file was copied from CoreRT. That part of the comment isn't supported by anything in the MSDN doc either. I'll remove it.